astacus/coordinator/plugins/__init__.py

```diff
@@ -1,10 +1,12 @@
 from .clickhouse.plugin import ClickHousePlugin
 from .files import FilesPlugin
+from .flink.flink import FlinkPlugin
```

Comment:
nit. Maybe renaming the file to `plugin.py` instead of having `flink.fink` ? (not critical, just for consistency)